python/ml4ir/applications/ranking/features/feature_fns/normalization.py

```diff
@@ -0,0 +1,72 @@
+import tensorflow as tf
+
+from ml4ir.base.features.feature_fns.base import BaseFeatureLayerOp
+from ml4ir.base.io.file_io import FileIO
+
+
+class TheoreticalMinMaxNormalization(BaseFeatureLayerOp):
+    """
+    Min Max Normalization of individual query features,
```

Comment:
```suggestion
    Min Max Scaling of individual query features,
```

Reply:
"Normalization" is still the correct usage, right?